static/js/zamboni/devhub.js

```diff
@@ -990,7 +990,10 @@ function initCatFields(delegate) {
             $("input", $misc).prop("checked", checkedLength <= 0);
         };
         var checkOther = function() {
-            $("input", $main).prop("checked", false).prop("disabled", false);
+          // Save Android categories
+          var $androidCat = $main.find('input[name="form-1-categories"]');
+          $("input", $main).prop("checked", false).prop("disabled", false);
+          $($androidCat+":checked").prop("checked", true).prop("disabled", false);
```

Comment:
I suspect this will be when `$androidCat` doesn't contain any matches - i.e. when there isn't a second form (called form-1-categories) because the add-on only supports one application.